### PR TITLE
feat(scoped-elements): load the scoped registries polyfill

### PR DESCRIPTION
**Pull request by manolakis**
## What I did

1. I removed the forked polyfill to use the webcomponents one.

### CODE REVIEW

File: packages/scoped-elements/package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
   "dependencies": {
     "@lit/reactive-element": "^1.0.0-rc.1",
     "@open-wc/dedupe-mixin": "^1.3.0",
-    "@webcomponents/scoped-custom-element-registry": "0.0.1"
+    "@webcomponents/scoped-custom-element-registry": "0.0.2"
   },
   "types": "types/index.d.ts",
   "sideEffects": [
```
Comment (lazka) [inline packages/scoped-elements/package.json:49]: The sideEffects array can be removed now I think

Comment (manolakis) [reply packages/scoped-elements/package.json]: That's right! thanks!

File: packages/scoped-elements/src/ScopedElementsMixin.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import './loadPolyfill.js';
+import '@webcomponents/scoped-custom-element-registry';
```
Comment (daKmoR) [inline packages/scoped-elements/src/ScopedElementsMixin.js:1]: does this now include the logic to not double load if it's executed 2 times from e.g. nested node_modules? 🤔

Comment (manolakis) [reply packages/scoped-elements/src/ScopedElementsMixin.js:1]: Yes, it includes all that logic plus all the fixes we send last month, even the logic to "upgrade" CE created before applying the polyfill

Comment (daKmoR) [reply packages/scoped-elements/src/ScopedElementsMixin.js:1]: sweet 💪

**Review by daKmoR — APPROVED** [review]

**Pull request merged**